recipe/meta.yaml

```
@@ -19,7 +19,7 @@ requirements:
     - pkg-config
     - make
   host:
-    - libfuse 2.*
+    - libfuse >=3
```

For better reproducibility of the the builds.
```suggestion
    - libfuse 3.*
```